site/en/docs/extensions/mv3/manifest/minimum_chrome_version/index.md

```diff
@@ -3,16 +3,26 @@ layout: "layouts/doc-post.njk"
 title: "Manifest - Minimum Chrome Version"
 seoTitle: "Chrome Extensions Manifest: minimum_chrome_version"
 date: 2013-05-12
-updated: 2023-01-04
+updated: 2023-02-14
 description: Reference documentation for the minimum_chrome_version property of manifest.json.
 ---
 
-An optional manifest key containing a string that defines what versions of Chrome are able to install the extension. The value set for this string must be identical to an existing Chrome browser version string. Either a full version number can be used if you need to specify a specific update to chrome, or simply the first number in the string for that version.
+An optional manifest key containing a string that defines what versions of Chrome are able to install the extension. The value set for this string must be a substring of an existing Chrome browser version string. Either a full version number can be used if you need to specify a specific update to Chrome, or simply the first number in the string to specify a particular major version.
```

Comment:
nit: I think we are now wrapping at 80 chars for these files.  Mind doing that, since we're updating them?